node/src/main/scala/org/bitcoins/node/PeerData.scala

```diff
@@ -26,7 +28,9 @@ case class PeerData(
       context = system,
       peer = peer,
       peerMessageReceiver = peerMessageReceiver,
-      onReconnect = node.sync
+      onReconnect = node.peerManager.onReconnect,
+      onStop = node.peerManager.onP2PClientStopped,
+      maxReconnectionTries = 4
```

Comment:
Changed max attempts to 4 as this would roughly be the time that we spend on each peer waiting for it to initialize. Ig all of these related constant need to be put together.